docs/helpers/each.md

```diff
@@ -14,7 +16,7 @@ is falsey or an empty list, render `INVERSE`.
 {{/each}}
 ```
 
-@param {can-stache/expressions/key-lookup|can-stache/expressions/call} EXPRESSION An
+@param {can-stache/expressions/key-lookup|can-stache/expressions/call|can-stache/expressions/hash} EXPRESSION An
```

Comment:
I think this needs to be a new signature for `{{#each EXPRESSION HASH_EXPRESSION}}FN{{else}}INVERSE{{/each}}`. 

`EXPRESSION` in `{{#each EXPRESSION }}` can't be a hash expression. That would be like `{{#each foo=bar baz=quz}}`.

Reply:
👍🏽